R/utils_dataprocess_plots.R

```diff
@@ -50,29 +49,34 @@
         input$is_censored = factor(input$is_censored, 
                                    levels = c("FALSE", "TRUE"))
     }
+    print(input)
+    print(unique(input$predicted))
+    write.csv(input, file = "your_file_name.csv", row.names = T)
     featureName = toupper(featureName)
     if (featureName == "TRANSITION") {
         type_color = "FEATURE"
     } else {
         type_color = "PEPTIDE"
     }
     
-    profile_plot = ggplot(input, aes_string(x = 'RUN', y = 'ABUNDANCE', 
+    profile_plot = ggplot(input, aes_string(x = 'RUN', y = 'newABUNDANCE',
```

Comment:
(usually) we use ", not '

Reply:
Commit [fix quotes](https://github.com/Vitek-Lab/MSstats/pull/107/commits/9992a05977fe9a2104f44cd2bb4fc97190e8f094)